etc/aim/aimctl.conf

```diff
@@ -0,0 +1,60 @@
+[DEFAULT]
+apic_system_id=openstack
+
+
+[apic]
+# The following snippet is derived from the APICAPI conf file
+# Hostname:port list of APIC controllers
+# apic_hosts = 1.1.1.1:80, 1.1.1.2:8080, 1.1.1.3:80
```

Comment:
Putting these options in the sample config file is just confusing. I think they should be removed.